docker_test

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
@evanjtravis please move `docker_test` to the `test` directory. Also I would prefer this as a Makefile in `test`, instead of a bash script.

Reply:
The script also does not build the Dockerfile. It needs too.

Reply:
Removed due to not being used/obsolete.